changelogs/fragments/432-build-release-role-minor-refactoring.yml

```
@@ -0,0 +1,5 @@
+---
+breaking_changes:
+ - The ``build-release`` role now depends on the ``community.general`` collection (https://github.com/ansible-community/antsibull/pull/432).
```

Make sure both indents are the same in this file :)
```suggestion
  - The ``build-release`` role now depends on the ``community.general`` collection (https://github.com/ansible-community/antsibull/pull/432).
```

---

Hmm, I must've made a mistake when I rebased this PR. I'll fix this later. I promise I know how to write proper yaml :).